common/python/schema_validation/WorkOrderGetResult.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Work Order Get Result Request Schema
+"""
+from jsonschema import validate, ValidationError, SchemaError
+
+
+def workorder_get_result_validation(params):
+    """
+    Validate params dictionary for existence of
+    mandatory fields in work order get result
+
+    Parameters:
+    params    Parameter dictionary to validate
+
+    Returns:
+    True and empty string on success and
+    False and string with error message on failure.
+    """
+
+    if len(params) == 0:
+        return False, "Work order Id not found"
+
+    schema = {
+        "type": "object",
+        "uniqueItems": True,
+        "additionalProperties": False,
+        "properties": {
+            "workOrderId": {
+                "type": "string",
+                "pattern": "^(0[x|X])?[0-9a-fA-F]+$",
+                "error_msg":
+                    "Invalid work order Id"
+                }
+        },
+        "required": ["workOrderId"]
+    }
+
+    try:
+        validate(params, schema)
+    except ValidationError as e:
+        if e.validator == 'additionalProperties' or \
+                e.validator == 'required':
+            return False, e.message
+        else:
+            return False, e.schema["error_msg"]
+    except SchemaError as err:
+        return False, err.message
+
+    return True, ""
```

Comment:
Please move schema loading from json and above code block to a base class `JsonSchemaLoader` for example.

Reply:
done